generated/subscription/microsoft.subscription/2021-10-01/types.md

```diff
@@ -11,7 +11,7 @@
 * **type**: 'Microsoft.Subscription/aliases' (ReadOnly, DeployTimeConstant): The resource type
 
 ## Resource Microsoft.Subscription/policies@2021-10-01
-* **Valid Scope(s)**: Tenant
+* **Valid Scope(s)**: Tenant, Extension
```

Comment:
There is [a read-only resource path defined as an extension on `Microsoft.Billing/billingAccounts` resources](https://github.com/Azure/azure-rest-api-specs/blob/10043a51add44d7249c2cec206f520d0fa64341a/specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json#L476-L509).

I don't believe we capture what resources and extension resource is allowed to extend in the scoping flags, so this is correct?